cdap-docs/integrations/source/apache-sentry.rst

```diff
@@ -8,6 +8,34 @@
 Apache Sentry
 =============
 
+.. _apache-sentry-configuration:
```

Comment:
shouldn't this stuff be in the sentry repo? Or why have the docs for sentry be in separate repo than the extension itself?